crates/ruma-common/src/events/room/member/change.rs

```diff
@@ -1,19 +1,18 @@
-use serde::{Deserialize, Serialize};
-
 use super::MembershipState;
 use crate::{MxcUri, UserId};
 
 /// The details of a member event required to calculate a [`MembershipChange`].
+#[derive(Clone, Debug)]
 pub struct MembershipDetails<'a> {
```

Comment:
I'm wondering why `clippy::exhaustive_structs` doesn't complain about this. Maybe a regression in Clippy? I'm not finding any higher-up `allow` attributes that would apply.